Assets/Scripts/System/SettingsSystem.cs

```diff
@@ -209,37 +209,19 @@ public SettingsSystem(ProjectSettingState projectSettingsState, SceneSettingStat
 
             var userSettings = new List<ISetting>();
 
-            TestNumber = new FloatUserSetting(this, "Test number", 12.34f);
-            userSettings.Add(TestNumber);
-
-            TestInteger = new IntUserSetting(this, "Test integer", 123);
-            userSettings.Add(TestInteger);
-
-            TestString = new StringUserSetting(this, "Test text", "Hello world!");
-            userSettings.Add(TestString);
-
-            applicationTargetFramerate = new IntUserSetting(this, "applicationTargetFramerate", -1);
-            userSettings.Add(applicationTargetFramerate);
+            applicationTargetFrameRate = new IntUserSetting(this, "Maximum frame rate", -1);
```

Comment:
The description "Maximum frame rate" did not end up in the final state of this branch.